miniapps/solvers/CMakeLists.txt

```diff
@@ -21,6 +21,12 @@ if (MFEM_USE_MPI)
     EXTRA_HEADERS lor_mms.hpp
     LIBRARIES mfem)
 
+  add_mfem_miniapp(lor_elast
+    MAIN lor_elast.cpp
+    EXTRA_SOURCES block_fespace_operator.cpp
+    EXTRA_HEADERS block_fespace_operator.hpp
+    LIBRARIES mfem)
+
```

Comment:
The new tests from the makefile should be added here as well

Reply:
I added the tests to cmake as well

[https://github.com/mfem/mfem/blob/4a38577e3f80d9252c1d81462a614a2206c4c3a6/miniapps/solvers/CMakeLists.txt#L54-L68](https://github.com/mfem/mfem/blob/4a38577e3f80d9252c1d81462a614a2206c4c3a6/miniapps/solvers/CMakeLists.txt#L54-L68)